modules/service/src/main/scala/lucuma/odb/service/ItcService.scala

```diff
@@ -295,10 +323,10 @@ object ItcService {
         itcInputs(params)
           .traverse { case (tid, input) => OptionT(selectSingleTarget(tid, input)) }
           .value
-          .map(_.map{ lst =>
-            val it: NonEmptyList[TargetResult] = lst.map(_._1)
+          .map(_.flatMap{ lst =>
```

Comment:
If we really only need to ask about the brightest target, we could go further here.  No need to call ITC for each target?  Maybe no need for `AsterismResult` at all.  This might simplify things a good bit.

Reply:
I like the warning idea.  A question is whether we can tell just based on the magnitudes that the asterism may be problematic or do we actually have to call the ITC and compare the exposure times for all the targets?

If we can get rid of `AsterismResult` it will ripple through the API but it would be a good simplification (and faster for multi-target asterisms) so I'm hopeful we can do it.

Reply:
Right, even if the calls are done in parallel they endup as an extra load on the ITC server